addons/point_of_sale/static/src/app/screens/receipt_screen/receipt/order_receipt.xml

```diff
@@ -138,4 +138,4 @@
             </div>
         </div>
     </t>
-</templates>
+</templates>
```

Comment:
add an empty line at the end. [ex for vscode](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files)